src/c-writer.cc

```diff
@@ -1516,13 +1516,20 @@ void CWriter::WriteElemInitializers() {
               GlobalName(elem_segment->name), "[] = ", OpenBrace());
         break;
       case Type::ExternRef:
-        Write("static const wasm_rt_externref_t elem_segment_exprs_",
```

Comment:
Why even declare this at all if its not going to be used?

Reply:
Reworked to eliminate this.